index.js

```diff
@@ -3,15 +3,25 @@
 var transpiler = require('babel-core');
 var Filter     = require('broccoli-filter');
 var clone      = require('clone');
+var mergeTrees = require('broccoli-merge-trees');
+var funnel = require('broccoli-funnel');
 
 function Babel(inputTree, options) {
   if (!(this instanceof Babel)) {
     return new Babel(inputTree, options);
   }
 
-  this.inputTree = inputTree;
   this.options = options || {};
   this.extensions = this.options.filterExtensions || ['js'];
+
+  if ( this.options.browserPolyfill ) {
+    delete this.options.browserPolyfill;
+    var src = 'node_modules/broccoli-babel-transpiler/node_modules/babel-core';
```

Comment:
you will likley need to use `require.resolve` to absorb dedupe based topology shifts in the node_module graph